src/components/Organisms/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useRef, useState, useEffect } from "react";
+import styled from "styled-components";
+import FONTFAMILY from "../../variables/font_family";
+import TEXTS from "../../variables/texts";
+import COLOR from "../../variables/color";
+import AddTaskButton from "../Atoms/AddTaskButton/index";
+import Tasks from "../Molecules/index";
+
+const TodoCard = () => {
+  const [taskList, setTaskList] = useState([{ name: "", initializing: true }]);
+
+  const onAddTaskButtonClick = () => {
+    setTaskList([...taskList,({ name: "", initializing: true })]);
+  };
+
+  const onTaskComplete = ({ index }) => {
+    const deleteList = [...taskList];
+    //console.log(index);
+    deleteList.splice(index, 1);
+    setTaskList(deleteList);
+    //setTaskList([[...taskList].slice(0, index), [...taskList].slice(index+1)])
+  };
+
+  const onTaskNameChange = ({ value, index }) => {
+    const changeList = [...taskList];
+    //console.log(value+" "+index);
+    if (value) {
+      //changeList[index].name = value;
+      changeList.splice(index, 1,{ name: value, initializing: true })
```

Comment:
```suggestion
      changeList.splice(index, 1,{ name: value, initializing: false })
```
`initializing` は、「最初から編集モードになっているかどうか」を管理したいので、一度編集したものは `false` にしたいです。